content/html/arm-request-hosting.html

```diff
@@ -0,0 +1,176 @@
+<div id="content">
+<!-- Formsender error script -->
+<script src="../../../theme/js/formsender-error.js"></script>
+  <div class="field field-name-body field-type-text-with-summary field-label-hidden">
+    <div class="field-items">
+      <div class="field-item even" property="content:encoded">
+        <p>The Open Source Lab partners with Ampere Computing to host ARM based servers in order to provide an open
+        platform for innovation to the open source community.<b>For more information about ARM Hosting, please visit
+        our <a href="/services/arm/">ARM Development Hosting page</a></b>.</p>
+        <p>Please use the form below to request <b>new</b> hosting on the ARM environment hosted at the OSUOSL.</p>
+        <p>For any <b>changes, updates, or issues with an existing project</b> please send the specific request as an
+          email to <a href="mailto:arm-hosting-request@osuosl.org">arm-hosting-request@osuosl.org</a> and include the
+          original project name in the subject line and the IP address(es) for the VM(s) in the message body of the
+          request which will help us narrow down what specific VM it's for.</p>
+        <p>This access is intended only for <b><i>free and open source</i></b> projects who qualify and are approved by
+          the OSUOSL. For proprietary sourced projects or products please TODO</p>
```

Comment:
@ramereth You may have missed a `TODO` here. What about the option for "proprietary sourced projects or products"?

Reply:
Yes I'm still waiting to hear back from them on what to put here.